### ISSUE TITLE
toolSettings.json gone in 15.3 preview

### ISSUE THREAD

**Issue by srivatsn** [aspnet/Tooling#1039]
_From @stgeorge on May 13, 2017 18:15_

After installing 2017 15.3 preview, I noticed that toolSettings.json is gone - can no longer customize the file nesting... but there is still file nesting going on, did this file move somewhere else or is something else managing the file nesting in 15.3 preview?

_Copied from original issue: dotnet/project-system#2167_

**Comment by srivatsn** [comment]
This was only implemented for web projects. Moving to aspnet/tooling to get an answer to this question.

**Comment by mlorbetske** [comment]
@BillHiebert or @abpiskunov can you provide more information here?

**Comment by BillHiebert** [comment]
It is in Common7\IDE\Extensions\Microsoft\Web Tools\ProjectSystem.  I looked at the changes and the code which used to generate the file was removed as part of fixing bug https://devdiv.visualstudio.com/web/wi.aspx?pcguid=011b8bdf-6d56-4f87-be0d-0092136884d9&id=383531

I suspect the reason was that the format was not locked down so we didn't want to allow user customization.

**Comment by abpiskunov** [comment]
Yes, Bill is right. We moved that file for now, and there is a pending workitem (currently for post 15.3) to enhance the design for customizing nesting, where we will finalize it. There is a TFS bug to track that this issue can be closed.

**Comment by mcgear** [comment]
Is there any update on this, or somewhere that it can be tracked?

**Comment by abpiskunov** [comment]
As i mentioned above it is tracked internally. We are planning to update nesting in the VS update that might happen in the end of fall.
